streams/src/main/java/org/apache/kafka/streams/kstream/WindowedSerdes.java

```diff
@@ -64,14 +64,6 @@ public SessionWindowedSerde(final Serde<T> inner) {
         return new TimeWindowedSerde<>(Serdes.serdeFrom(type));
     }
 
-    /**
-     * Construct a {@code TimeWindowedSerde} object to deserialize changelog topic
-     * for the specified inner class type and window size.
-     */
-    static public <T> Serde<Windowed<T>> timeWindowedSerdeFrom(final Class<T> type, final long windowSize) {
```

Comment:
This is public API and cannot just be removed. Why do you think it should be removed?

Reply:
I'm sorry, it was a mistake; So reverted.

Reply:
@dongjinleekr Seems you did not revert this change yet.